Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM debian:wheezy
+FROM debian:bookworm-slim
```

Comment:
Yes, it's about time to update :)

Could you update `README.md` accordingly as well?